packages/http-client-python/emitter/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./emitter.js";
```

Comment:
```suggestion
export {$onEmit} from "./emitter.js";
```
just so you only export what you need